ci/docker/runtime_functions.sh

```diff
@@ -806,6 +806,24 @@ build_ubuntu_gpu_cuda101_cudnn7() {
     make cython PYTHON=python3
 }
 
+build_ubuntu_gpu_cuda110_cudnn8() {
+    set -ex
+    build_ccache_wrappers
+    make \
+        USE_BLAS=openblas                         \
+        USE_MKLDNN=0                              \
```

Comment:
MKLDNN is ON by default right?

Reply:
yes, but this particular job compiled with it explicitly disabled

Reply:
I copied the build function from the main full-gpu build (see https://github.com/apache/incubator-mxnet/blob/v1.x/ci/jenkins/Jenkinsfile_unix_gpu#L39) since it seems like we wanted a slimmed-down pipeline for cuda 11.0 for now.